frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseResetBranchToRemoteAction.java

```diff
@@ -46,7 +47,7 @@ public IEnhancedLambdaLogger log() {
 
   @Override
   public String getActionName() {
-    return "Reset";
+    return "Reset to Remote";
```

Comment:
Given how it's later used in the descriptions... shouldn't it rather be lower case `"Reset to remote"`?